src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -418,6 +418,13 @@ pub struct Args {
         default_value = "http://localhost:6879"
     )]
     persist_pubsub_url: String,
+    /// Whether to use the new persist-txn tables implementation or the legacy
+    /// one.
+    ///
+    /// TODO(txn): Figure out how to configure this via LaunchDarkly instead of
+    /// a flag.
+    #[clap(long, env = "ENABLE_PERSIST_TXN_TABLES")]
+    enable_persist_txn_tables: bool,
```

Comment:
note to self: we should plumb this to clusterd processes so that everything restarts when the flag is flipped on envd (as is necessary)